src/admin/migrations/m210605_172811_cms_website_permissions.php

```diff
@@ -13,8 +13,8 @@ class m210605_172811_cms_website_permissions extends Migration
      */
     public function safeUp()
     {
-        $this->addColumn(Website::tableName(), 'group_ids', $this->text());
-        $this->addColumn(Website::tableName(), 'user_ids', $this->text());
+        $this->addColumn(Website::tableName(), 'group_ids', $this->text()->defaultValue('[{"value":0}]'));
+        $this->addColumn(Website::tableName(), 'user_ids', $this->text()->defaultValue('[{"value":0}]'));
```

Comment:
What does `['value' => 0]` mean? When there is "real" data it would be `['value' => [1,2,3]` where ids are real users in array annotation?

Reply:
I am askin because; when you select users in de admin UI the 0 will be gone, but how to restore this default value again? there must be a checkbox like "everyone is permitted" - otherwise "no selected users" can be equal to "all have been selected". do you know what i mean?

Reply:
Or will the `['value' => 0]` automatically restored when no users/groups are selected? i don't think so? otherwise it would not be possible to deny permission to all users/groups.